swagger2.0/oic.r.grouping.json

```diff
@@ -0,0 +1,124 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Grouping",
+    "version": "2021-07-15",
+    "license": {
+      "name": "OCF Data Model License",
+      "url": "https://github.com/openconnectivityfoundation/core/blob/e28a9e0a92e17042ba3e83661e4c0fbce8bdc4ba/LICENSE.md",
+      "x-copyright": "Copyright 2021 Open Connectivity Foundation, Inc. All rights reserved."
+    },
+    "termsOfService": "https://openconnectivityfoundation.github.io/core/DISCLAIMER.md"
+  },
+  "schemes": ["http"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/groupsResURI" : {
+      "get": {
+        "description": "The Resource to set groups to which the device belongs to. To be used in conjuction with SSM.\n",
```

Comment:
Grammar and spelling fixes:
"The Resource to set groups to which the device belongs. To be used in conjunction with SSM (Simple Secure Multicast)."

Reply:
fixed